framework/include/auxkernels/TagAuxBase.h

```diff
@@ -50,16 +50,23 @@ TagAuxBase<T>::validParams()
                                "The coupled variable whose components are coupled to AuxVariable");
   params.set<ExecFlagEnum>("execute_on", true) = {EXEC_TIMESTEP_END};
   params.suppressParameter<ExecFlagEnum>("execute_on");
-  params.addParam<bool>("scaled",
-                        true,
-                        "Return value depending on the variable scaling/autoscaling. Set this to "
-                        "false to obtain unscaled physical reaction forces.");
+  params.addDeprecatedParam<bool>(
+      "scaled",
+      true,
+      "Return value depending on the variable scaling/autoscaling. Set this to false to obtain "
+      "unscaled values with units consistent with the rest of the simulation.",
+      "This parameter has been replaced by remove_variable_scaling.");
+  params.addParam<bool>(
+      "remove_variable_scaling", false, "If true, the variable scaling will be removed.");
```

Comment:
We suppress this in the `TagSolutionAux` class and we do not yet have an implementation for this parameter in the tagged matrix aux kernel. I think this should be moved to `TagResidualAux`

Reply:
That's true.